### PR TITLE
Enable row-level-security on local dev environment

### PR DESCRIPTION
**Pull request by Elkasitu**
This commit bridges the gap between local and stage/prod environments by enabling RLS by default on **new** local dev environment databases, this means that testing out things like performance, application of data migrations and general workflows should be closer to the aforementioned environments.

Closes OSIDB-264

### CODE REVIEW

**Review by osoukup — APPROVED** [review]
LGTM. Nice work!

File: osidb/tests/test_rls.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+from osidb.core import set_user_acls
+from osidb.models import Flaw
+from osidb.tests.factories import FlawFactory
+
+pytestmark = pytest.mark.enable_rls
+
+
+class TestRLS:
+    # XXX: The following tests use Flaw for a model to test ACLs on,
```
Comment (jsvob) [inline osidb/tests/test_rls.py:14]: Leftover `XXX`?

Comment (Elkasitu) [reply osidb/tests/test_rls.py:14]: intended, it's just documentation

Comment (osoukup) [reply osidb/tests/test_rls.py:14]: Not sure whether it is a convention but at least my editor (Vim) highlights **XXX** string by default (the same ways is it does for **TODO**).

Comment (jobselko) [reply osidb/tests/test_rls.py:14]: Yeah, it is used very often and it means sth like `TODO`/`FIX`/`NOTE`.

**Pull request merged**